adventurine.rb

```diff
@@ -1,26 +1,87 @@
 require 'pp' #so the board prints prettier
 
 board = [
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"]
 ]
 
-user_pos = [0,0]
-a = [-1,0]
-s = [0,1]
-w = [0,-1]
-d = [1,0]
+user_x = 0 
+user_y = 0
+valid_moves = ['w', 'a', 's', 'd', 'x', 'p']
 
-pp board
-print "You are in the northwestern most corner of a dungeon. Use 'a' to move west, 's' to move south, 'w' to move north, and 'd' to move east: "
-user_input = gets.chomp.downcase
+puts "What is your character's name?"
+character = gets.chomp
```

Comment:
Also unnecessary...I just did this so that the cursor moving around came from a user-inputted name. Could have just as easily been a fixed character